ccm_web/Dockerfile.openshift

```diff
@@ -0,0 +1,32 @@
+# Build stage (build client and compile server to JS)
+FROM docker-registry.default.svc:5000/openshift/node:14-slim AS build
+WORKDIR /build/
+
+COPY package*.json ./
+RUN npm install
+
+COPY . .
+RUN npm run build
+
+# Prod stage
+FROM docker-registry.default.svc:5000/openshift/node:14-slim AS prod
+ENV NODE_ENV=production
+WORKDIR /app
+
+COPY --from=build \
+    /build/package.json \
+    /build/package-lock.json \
+    /build/start.sh \
+    /build/ecosystem.config.js \
+    ./
+RUN npm install --production
+RUN npm install pm2@5.1.0 -g
+COPY --from=build /build/dist/ ./
```

Comment:
Reference `--from=0` instead of `--from=build`?

Reply:
Didn't know [it numbered the FROMs.](https://docs.docker.com/develop/develop-images/multistage-build/#name-your-build-stages)

I'd agree with Chris that if it's working don't change anything with this now. This isn't something going away in future 3.x releases.